Fix the dateSerializationFormat description (#2571)
DevExpress/devextreme-documentation: includes/uiwidgets-ref-dateSerializationFormat.md

Specify this property in the following cases:

> - The [dataSource](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/dataSource.md '{basewidgetpath}/Configuration/#dataSource') is empty or not set at design time. The **dateSerializationFormat** is needed because the {WidgetName} cannot detect it automatically without a data source.

Reviewer: ```suggestion
- The [dataSource](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/dataSource.md '{basewidgetpath}/Configuration/#dataSource') is empty or not set at design time. The **dateSerializationFormat** is needed, because the {WidgetName} cannot detect it automatically without a data source.
```

> - You use the <a href="https://github.com/DevExpress/DevExtreme.AspNet.Data/blob/master/docs/client-side-with-jquery.md#api-reference" target="_blank">createStore</a> method from the <a href="https://github.com/DevExpress/DevExtreme.AspNet.Data#devextreme-aspnet-data" target="_blank">DevExtreme.AspNet.Data</a> extension and date-time values that come from the server are specified in UTC. DevExtreme.AspNet.Data cannot correctly serialize UTC values because it uses a different date-time serialization format.

Reviewer 2: ```suggestion
- You use the <a href="https://github.com/DevExpress/DevExtreme.AspNet.Data/blob/master/docs/client-side-with-jquery.md#api-reference" target="_blank">createStore</a> method from the <a href="https://github.com/DevExpress/DevExtreme.AspNet.Data#devextreme-aspnet-data" target="_blank">DevExtreme.AspNet.Data</a> extension to create a custom remote data store and its date-time values use the UTC format. In this case, DevExtreme.AspNet.Data requires a specified **dateSerializationFormat** to serialize UTC values.
```

> "You use the createStore method from the DevExtreme.AspNet.Data extension and date-time values that come from the server are specified in UTC."

It is not explicit that these "date-time values" come as a result of using createStore.

> "DevExtreme.AspNet.Data cannot correctly serialize..."

Is it better to write about the solution ("specified dateSerializationFormat" from the suggestion) rather than the problem ("DevExtreme.AspNet.Data cannot correctly serialize")?

Author: > It is not explicit that these "date-time values" come as a result of using createStore.

They don't. The dateSerializationFormat is required when both of these conditions are met: 1) createStore is used; 2) dates on the server are in UTC.

```suggestion
- You use the <a href="https://github.com/DevExpress/DevExtreme.AspNet.Data/blob/master/docs/client-side-with-jquery.md#api-reference" target="_blank">createStore</a> method from the <a href="https://github.com/DevExpress/DevExtreme.AspNet.Data#devextreme-aspnet-data" target="_blank">DevExtreme.AspNet.Data</a> extension and remote date-time values are specified in UTC. DevExtreme.AspNet.Data requires the **dateSerializationFormat** to correctly serialize these values.
```

Use one of the following values to specify the **dateSerializationFormat** property:

> - `"yyyy-MM-dd"` - a local date  

Reviewer: ```suggestion
- `"yyyy-MM-dd"` - local date  
```
